pytest

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python
+
```

Comment:
Would it be possible to rather solve that through conftest.py or is that too late for monkey patching. Needing to run this file instead of pytest is not so nice I think.

Reply:
It might be possible or it might already be too late. I'd rather err on the safe side and put this wrapper around pytest here.